packages/core/botpress/src/botpress.js

```diff
@@ -122,6 +130,11 @@ const validateBotfile = botfile => {
     throw new Error('Outdated botfile format')
   }
 
+  const { version } = botfile
+  const validateVersion = compose(isEqualOrGreater(packageJson.version), isValidFormat, isValidString, existVersion)
```

Comment:
Why don't we export it as a single function from `utils` if all the mentioned functions are only used once, and only here?

Reply:
@emirotin I don't want pollute the `botpress.js` but I can change

Reply:
You don't need to pollute anything, I'm saying all these methods are only used as part of `validateVersion` so you could _only_ export this assembled method from utils